app/components/layout/Navigation/MyBuilds/index.js

```diff
@@ -16,29 +17,46 @@ import CachedStateWrapper from '../../../../lib/CachedStateWrapper';
 import Build from './build';
 import DropdownButton from './../dropdown-button';
 
-class MyBuilds extends React.Component {
-  static propTypes = {
-    viewer: PropTypes.object,
-    relay: PropTypes.object.isRequired
-  }
+type Props = {
+  viewer?: Object,
+  relay: Object
+};
 
+type State = {
+  isDropdownVisible: boolean,
+  scheduledBuildsCount: number,
+  runningBuildsCount: number
+};
+
+class MyBuilds extends React.Component<Props, State> {
```

Comment:
This typing pattern is neat! I hadn't seen it before. 👌

Reply:
It’s great! Flow is *really* flexible, but provides really valuable checks I’d love to get more coverage out of across the codebase!

Reply:
💯